PCP/src/main/java/PCP/logic/IMemoryAccess.java

```diff
@@ -7,7 +7,7 @@
 
 
 /**
- * interface to allow memory access
+ * base interface for memory access managmenet
```

Comment:
base interface for memory access **managemenet**

Reply:
I know, it's bullshit

Reply:
No no it's ok 😆 
You could actually fix that yourself with a commit. Imma focus on other things.